sdk/core/core-amqp/src/errors.ts

```diff
@@ -614,7 +633,7 @@ export function translate(err: AmqpError | Error): MessagingError {
 
   if (isBrowserWebsocketError(err)) {
     // Translate browser communication errors during opening handshake to generic SeviceCommunicationError
-    error = new MessagingError("Websocket connection failed.");
+    const error = new MessagingError("Websocket connection failed.");
     error.name = ConditionErrorNameMapper[ErrorNameConditionMapper.ServiceCommunicationError];
```

Comment:
Shouldnt this be code?
```suggestion
    error.code = ConditionErrorNameMapper[ErrorNameConditionMapper.ServiceCommunicationError];
```

Reply:
Good catch! Will update.